charts/localstack/values.yaml

```diff
@@ -173,6 +173,10 @@ resources: {}
   # requests:
   #   cpu: 100m
   #   memory: 128Mi
+resources:
+  requests:
+    cpu: 1000m
+    memory: 4Gi
```

Comment:
The values here in the `values.yaml` are default values used for every single deployment, not only for those on EKS.
Your change here sets it for everyone, where I would prefer to have this in a documentation, or in a shared values file specifically for EKS which can then be used when deploying to EKS.
For example:
```
helm install -f eks-values.yaml localstack/localstack
```
If these settings are really applicable for _every_ EKS cluster, we could share them here directly, or we could put them on the docs pages.

But actually, I am not sure why these are necessary in the first place. Could you maybe please add some explanation why it is necessary to limit the CPU and memory in EKS?

Reply:
Empty defaults are too low. The container fails to come up, runs out of memory and is killed. I think not setting the values at all leaves us with a LocalStack that won't run in most cases. Should we have a base default like this? Or always force an override?

Reply:
@alexrashed these are not limits, but _minimum_ requirements. I don't know if `k3s` respects resource requests like this, but EKS with fargate certainly requires them.

I like the idea of having example "supported" override files for different scenarios, however I suspect there are only two, but there may be more:

* local using k3d, the default with LocalStack where it doesn't matter since the default nodes have the resources of the host, and
* a "real" k8s cluster e.g. EKS where we have to bump up the values a bit.